client/src/components/Item/item.style.js

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+
+export const ItemContainer = styled.div`
+  width: 80%;
+  height: 100px;
+  box-shadow: 1px 2px 2px rgba(195, 184, 184, 0.25);
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const ItemDiv = styled.div`
+  margin-top: 20px;
+  display: flex;
+  justify-content: center;
+}`;
+
+export const ItemImage = styled.img`
+  height: 100px;
+  width: 30%;
+}`;
+
+export const Line = styled.div`
+  height: 100px;
+  width: 5px;
+  background-color: #1ed390;
+  margin-right: 10px;
+`;
+
+export const ItemTitle = styled.p`
+  font-family: Assistant;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: normal;
+  color: #000000;
+`;
+export const ItemText = styled.p`
+  margin-left: 3px;
+  font-family: Assistant;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: normal;
+  color: #4c5268;
+`;
+
+export const ItemMainData = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+`;
+
+export const ItemSubtitle = styled.p`
+  font-family: Assistant;
```

Comment:
This is already in the global css body. This comment is for all the font family declarations.

Reply:
removed